Chat App/View/Pub/PubChatLoginViewController.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  PubChatLoginViewController.swift
+//  Chat App
+//
+//  Created by Amiel Jireh Cordova on 12/7/20.
+//  Copyright © 2020 Cordova, Jireh. All rights reserved.
+//
+
+import UIKit
+
+class PubChatLoginViewController: UIViewController {
+    
+    var bannerLabel: UILabel!
```

Comment:
PP: lazy var bannerLabel: UILabel = UILabel()

Reply:
Resolved in the next commit.